macros/schema_tests/aggregate_functions/expect_column_stdev_to_be_greater_than.sql

```diff
@@ -0,0 +1,42 @@
+{% macro test_expect_column_stdev_to_be_greater_than(model, 
+                                                     column_name,
+                                                     value,
+                                                     group_by=None,
+                                                     row_condition=None) -%}
+
+    {{ adapter.dispatch('test_expect_column_stdev_to_be_greater_than', 
+        packages = dbt_expectations._get_namespaces()) (model, 
+                                                        column_name,
+                                                        value,
+                                                        group_by,
+                                                        row_condition) }}
+{%- endmacro %}
+
+{% macro default__test_expect_column_stdev_to_be_greater_than(model, 
+                                                              column_name,
+                                                              value,
+                                                              group_by,
+                                                              row_condition) %}
+
+with calc as (
+    select distinct
```

Comment:
I don't think we need that `distinct` anymore?

Reply:
Good catch.